### PR TITLE
Fix normalization integration tests in CI

### PR DESCRIPTION
**Pull request by ChristopheDuong**
## What
GitHub CI fails to run normalization integration tests successfully because it can't find and build docker images as needed

## How
*Describe the solution*

### CODE REVIEW

File: tools/bin/ci_integration_test.sh [hunk 1]
```diff
@@ -13,7 +13,17 @@ if [[ "$connector" == "all" ]] ; then
   echo "Running: ./gradlew --no-daemon --scan integrationTest"
   ./gradlew --no-daemon --scan integrationTest
 else
-  if [[ "$connector" == *"connectors"* ]] || [[ "$connector" == *"bases"* ]]; then
+  if [[ "$connector" == *"base-normalization"* ]]; then
+    selected_integration_test="base-normalization"
+    integrationTestCommand="$(_to_gradle_path "airbyte-integrations/bases/base-normalization" integrationTest)"
+    export SUB_BUILD="NORMALIZATION"
+    ./gradlew --no-daemon --scan airbyteDocker
```
Comment (ChristopheDuong) [inline tools/bin/ci_integration_test.sh:20]: I don't quite understand why `:airbyte-integrations:bases:base-normalization:airbyteDocker` is not executed when calling `:airbyte-integrations:bases:base-normalization:integrationTest`

even though `customIntegrationTestPython.dependsOn ':airbyte-integrations:bases:base-normalization:airbyteDocker'` ...

So to make the tests passed in the CI, I couldn't figure out anything other than explicitly invoking it here... maybe there's a better way?

**Review by marcosmarxm — APPROVED** [review]
Great! Do you think we need to investigate further why this happened?

File: tools/bin/ci_integration_test.sh (hunk 1 above)
Comment (davinchia) [reply tools/bin/ci_integration_test.sh:20]: yeah I remember looking at this with you and being surprised too.

this seems 'ok' as a short term fix. we should definitely create a follow up issue and look into why this is happening when we have time (after Beta launch).

**Review by davinchia — APPROVED** [review]

**Pull request merged**